scripts/dev/setup_kind_cluster.sh

```diff
@@ -29,3 +29,17 @@ containerdConfigPatches:
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
     endpoint = ["http://${ip}:${reg_port}"]
 EOF
+
+# Document the local registry
+# https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/generic/1755-communicating-a-local-registry
```

Comment:
Cool, I didn't realize about the new required configuration.

Can you point to this page instead? That's the one corresponding to Kind in particular.

https://kind.sigs.k8s.io/docs/user/local-registry/